trackmania2020/egg-trackmania2020.json:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://pelican.stather.me/

```

Comment:
what is this?

Reply:
huh, it seems to be a meta data file that windows hides, I copied the egg.json into WSL env to use git and windows seems to have copied that file with it. I have deleted it.